lib/identity_cache/query_api.rb

```diff
@@ -106,9 +106,16 @@ def prefetch_associations(associations, records)
         nil
       end
 
-      def record_from_coder(coder) #:nodoc:
-        return nil unless coder
+      def instrumented_record_from_coder(coder) #:nodoc:
+        return unless coder
```

Comment:
Nitpick: I would prefer `return nil` over `return` when the return value is meant to be used.